.github/workflows/publish-results.yml

```
@@ -7,8 +7,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
-      - uses: actions/setup-python@v4
+        uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
         with:
           python-version: '3.11'
       - name: Install Requirements
```

```suggestion
      - name: Install Requirements
        run: python -m pip install --requirement requirements.txt
```
See above.
I wasn't able to select the line below in the suggestions, but you can delete it.